.github/actions/get-current-time/action.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Get Current Time Stamp'
+description: 'Get the current time stamp'
+
+outputs:
+  timestamp:
+    description: "new Date().toString()"
+
+runs:
+  using: 'node14'
```

Comment:
Can you bump up **all** the node14 to node18 while you're at it? Across all the workflows.

Reply:
was seeing some issues after converting to run on node18, we should save this for a separate effort.